lib/styles.scss

```diff
@@ -66,16 +75,21 @@ h2  {
   display: flex;
   justify-content: space-around;
 }
+.hourlyTemps {
+  border-bottom: 1px solid white;
+}
 
 .dayContainer, .precipContainer, .precipTypeContainer, .highContainer, .lowContainer {
   display: flex;
   flex-direction: column;
 }
 .todaySummary {
   font-size: 1.5rem;
+  font-family: Arial;
 }
 .dailyData {
   display: flex;
   justify-content: space-around;
   font-size: 1.5rem;
+  font-family: Arial;
```

Comment:
I like the colors! If you add font-family: Arial to the body, then you do not have type it every time for each container! It also might be nice if you only underlined the font on the right hand box instead of the whole line, but that is a design decision!

Reply:
Great catch!  Thank you!